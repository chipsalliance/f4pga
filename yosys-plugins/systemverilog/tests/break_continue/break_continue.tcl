yosys -import
if { [info procs read_uhdm] == {} } { plugin -i systemverilog }
yosys -import  ;# ingest plugin commands

set TMP_DIR /tmp
if { [info exists ::env(TMPDIR) ] } {
  set TMP_DIR $::env(TMPDIR)
}

# Testing simple round-trip
read_systemverilog -o $TMP_DIR/break-continue-test $::env(DESIGN_TOP).v
prep
write_table $::env(DESIGN_TOP).out
