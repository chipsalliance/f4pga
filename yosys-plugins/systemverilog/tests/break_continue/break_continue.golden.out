top	a	-	-	po	110
top	b	-	-	po	15
